src/lib/Webhook.coffee

```diff
@@ -0,0 +1,61 @@
+request = require "requestretry"
+
+DEFAULT_TIMEOUT = 10 * 1000
+DEFAULT_MAX_ATTEMPTS = 3
+
+class Webhook
+
+  constructor: (options = {}) ->
+    @options = options
+    @timeout = DEFAULT_TIMEOUT
+    @maxAttempts = DEFAULT_MAX_ATTEMPTS
+
+  @detectEmoji: (emoji) ->
```

Comment:
can you write a short doc here describing the input and response values? what's `val`, what is `emoji` supposed to be?